.scalafmt.conf

```diff
@@ -0,0 +1,63 @@
+version = 3.3.0
```

Comment:
Relatively unimportant, but how many of the non-defaults are necessary?  One of the virtues of a formatter is minimizing trivial differences in the wild.  The closer this hews to upstream defaults, generally the better.

(I make an exception on vertical alignment because of merges.  I don't claim to not be a hypocrite.)

Reply:
I don't have any attachment to these particular settings, I just like auto-formatting :) I copy this config around my projects. I think it originated in CE3.

(In case it's not clear, this is only for this repo—this plugin doesn't dictate anything about formatting.)

Reply:
Yeah, let's not make this a distraction.  Anyway, I like slim ones in the spirit of having a more standardized dialect of Scala.